app/views/submissions/_form.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<%= bootstrap_form_for [@project, @submission] do |form| %>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        New Job Submission
+      </div>
+
+      <div class="panel-body", role="main">
+        <%= form.text_field :name, required: true %> 
+        <%= form.text_field :frames, required: true %>
+        <%= form.text_field :camera, required: true %> 
+        <%= form.text_field :file, required: true, 
+          'data-filepicker': @project.directory %>
+        
+        
+        <%= form.select :cluster, OODClusters.map { |cluster| [ "#{cluster.metadata.title}", cluster.id ] }, 
+            { label: "Cluster:", :include_blank => true }, { class: "selectpicker", id: "batch_host_select" } %>
+
+
+        <button type="button" name="show_extra_vars" 
+                class="btn btn-primary btn-sm" onclick="toggle('extra_form_items')">More Options</button>
+            <br>
+
+        <div id="extra_form_items" style="display: none;">
+          <%= form.select :renderer, 
+            ['arnold','default','file','hw2','hw','interBatch','sw','turtle','turtlebake','vr'], required: true %>
+          <%= form.text_field :extra, required: false %>
+          <%= form.number_field :cores, required: true %>
```

Comment:
remove the option to change cores at this time; since a default value is set just removing this line should be sufficient